maps/away/ships/kataphracts/kataphract_ship.dmm

```diff
@@ -4300,14 +4299,14 @@
 	icon_state = "corner_white";
 	dir = 5
 	},
+/obj/machinery/atmospherics/pipe/simple/hidden/supply{
+	dir = 4
+	},
 /obj/structure/cable{
-	d1 = 1;
+	d1 = 2;
```

Comment:
don't manually set the d1 and d2 values, the code handles this. set the icon_state to the direction you want to use.

![cabletypes](https://user-images.githubusercontent.com/99297919/190399869-e46104e5-bcc4-4dbd-9af1-97e1fcf694dc.png)

Reply:
Weird part to me is, that's something I pasted in from the aurora/horizon rather than manually set a var for (a lot of old away site wires break themselves),

Reply:
Keeps breaking when I touch it and seems to work fine as is either way.